scalableminds/webknossos
# Rework volume interpolation feature to be shortcut-bound and support depths > 2

From changelog:
- Added support for segment interpolation with depths > 2. Also, the feature was changed to work on an explicit trigger (either via the button in the toolbar or via the shortcut V). When triggering the interpolation, the current segment id is interpolated between the current slice and the least-recently annotated slice.
- Removed the feature to copy a segment from the previous/next slice with the V shortcut. Use the new volume interpolation feature instead (also bound to V and available via the toolbar).


### URL of deployed dev instance (used for testing):
- https://deepinterpolation.webknossos.xyz/

### Steps to test:
- create a new annotation
- label a cell in slice X
- move forward to slice X + 3
- press "V" shortcut --> X + 1 and X + 2 should be filled automatically
- repeat the same with varying constellations (e.g., z distance == 1 which should disable the button and show a toast when pressing V; distance > 8; interpolating for a slice where the segment does not exist)

### Issues:
- fixes #6183 

------
(Please delete unneeded items, merge only when none are left open)
